### PR TITLE
5-js-logic elena_ambitska 

### PR DESCRIPTION
**Pull request by elena-ambitska**
Yet another calculator :)

### CODE REVIEW

File: homeworks/taras.bobov_LotusElize/GamesProject/success.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <!-- <link rel="stylesheet" href="css/style.css"> -->
+        <!-- <link rel="stylesheet" href="css/styles.css"> -->
```
Comment (vladyslav-yermolin-moc) [inline homeworks/taras.bobov_LotusElize/GamesProject/success.html:6]: Ouch, why do you like to edit files of another student?

File: lectures/5-logic/script.js [hunk 2]
```diff
@@ -2,12 +2,12 @@ const FIRST_VALUE_ELEMENT = document.getElementById('firstValue');
 const SECOND_VALUE_ELEMENT = document.getElementById('secondValue');
 const OPERATION_ELEMENT = document.getElementById('operation');
 const CALCULATE_BUTTON_ELEMENT = document.getElementById('calculate');
-const RESULT_ELEMENT = document.getElementById('result');
+const IMAGE_ELEMENT = document.getElementById('result');
```
Comment (vladyslav-yermolin-moc) [inline lectures/5-logic/script.js:5]: Here also. 
Would be very useful to check your pr files before sending the pr to review
image

File: homeworks/olena.ambitska_elena-ambitska/5-logic/homework.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+ * - if the result is more than 100, return 'Result is too big'
+ * - else return number of result: 1 '+' 2 returns 3
+ *
+ */
+
+function calculate(firstValue, secondValue, operation) {
+    let result;
+
+    if (Number.isNaN(firstValue) || Number.isNaN(secondValue)) {
+        return 'Enter a number';
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olena.ambitska_elena-ambitska/5-logic/homework.js:18]: Should be an error for this case 
image

File: homeworks/olena.ambitska_elena-ambitska/5-logic/index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                    <option value="*">*</option>
+                    <option value="**">**</option>
+                </select>
+            </label>
+            <label>
+                <input id="secondValue" type="number" min="0" step="1" class="calc__input">
+            </label>
+            <button id="calculate" type="button" class="calc__button">=</button>
+        </div>
+        <section class="counter__section">
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olena.ambitska_elena-ambitska/5-logic/index.html:37]: Hey, @elena-ambitska, so good so far. Some comments from me:
1. For `*` task, the calculator page should be integrated into your project.
2. Please align pictures with inputs.
3. Place  the resulting cards in the column of 10 pcs

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

File: homeworks/taras.bobov_LotusElize/GamesProject/success.html (hunk 1 above)
Comment (elena-ambitska) [reply homeworks/taras.bobov_LotusElize/GamesProject/success.html]: @vladyslav-yermolin-moc My fault. Sorry. Auto-refactor plus my inattention :(

File: lectures/5-logic/script.js (hunk 2 above)
Comment (elena-ambitska) [reply lectures/5-logic/script.js]: @vladyslav-yermolin-moc fixed

Comment (elena-ambitska) [reply lectures/5-logic/script.js]: @vladyslav-yermolin-moc agreed with you. accepted

File: homeworks/olena.ambitska_elena-ambitska/5-logic/homework.js (hunk 3 above)
Comment (elena-ambitska) [reply homeworks/olena.ambitska_elena-ambitska/5-logic/homework.js]: @vladyslav-yermolin-moc done
<details>

image

 </details>

File: homeworks/olena.ambitska_elena-ambitska/5-logic/index.html (hunk 4 above)
Comment (elena-ambitska) [reply homeworks/olena.ambitska_elena-ambitska/5-logic/index.html]: @vladyslav-yermolin-moc done for every thread

<details>

Screenshot 2023-11-04 at 15 43 33

</details>

Comment (vladyslav-yermolin-moc) [reply homeworks/olena.ambitska_elena-ambitska/5-logic/index.html]: Please align the columns 
image

Comment (vladyslav-yermolin-moc) [reply homeworks/olena.ambitska_elena-ambitska/5-logic/index.html]: Please move your files to the project folder 
image

Comment (vladyslav-yermolin-moc) [reply homeworks/olena.ambitska_elena-ambitska/5-logic/index.html]: Please update the header
image

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]
the thread with recommendation https://github.com/MastersAcademy/fe-course-basic-2023/pull/278#discussion_r1382448070

File: homeworks/olena.ambitska_elena-ambitska/5-logic/index.html (hunk 4 above)
Comment (elena-ambitska) [reply homeworks/olena.ambitska_elena-ambitska/5-logic/index.html]: @vladyslav-yermolin-moc done 

image

File: homeworks/olena.ambitska_elena-ambitska/GamesProject/5-logic/calc.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
```
Comment (vladyslav-yermolin-moc) [inline homeworks/olena.ambitska_elena-ambitska/GamesProject/5-logic/calc.html:1]: I've meant the full integration without actually `5-logic` folder. The html file is near all other html files, css - to style folder, js - to srcipt folder, probably.
Ok, I'm merging, you can fix in the next task.

**Review by vladyslav-yermolin-moc — APPROVED** [review]

File: homeworks/olena.ambitska_elena-ambitska/GamesProject/5-logic/calc.html (hunk 5 above)
Comment (elena-ambitska) [reply homeworks/olena.ambitska_elena-ambitska/GamesProject/5-logic/calc.html:1]: @vladyslav-yermolin-moc  thanks. I will fix this in the next PR

**Pull request merged**